### PR TITLE
Updated Website

### PR DESCRIPTION
**Pull request by gu-ana**
Added Rishi and Justin's updated bio.
Added Current Track's times (do not have a time and date for Advanced Track)
Added a "past events" on the bottom of /events, let students see what we have done in the past.

To do:
ICPC student article when it comes out.
updated officer bios
updated descriptions of events (which room, times, etc.)

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -1,58 +1,71 @@
...
+    day: 'Thu',
+    date: 'Oct. 7',
     time: '6 pm - 8 pm PT',
-    location: 'Zoom',
-    description: 'Trees and Graphs',
-    facebook: 'https://fb.me/e/5DhHuB1F5',
-    discord: 'https://discord.com/channels/702801010426511373/713559003661336607',
+    location: 'In Person!',
+    description: 'Intro to Coding Interviews/Arrays \n More details soon.',
+    // facebook: 'https://fb.me/e/5DhHuB1F5',
```
Comment (entrohpy) [inline src/data.js:25]: would suggest just linking to the ACM ICPC facebook page and `#icpc` discord channel

File: src/data.js [hunk 2]
```diff
@@ -1,58 +1,71 @@
...
+    name: 'Coding Interview #1',
+    day: 'Thu',
+    date: 'Oct. 7',
     time: '6 pm - 8 pm PT',
-    location: 'Zoom',
-    description: 'Trees and Graphs',
-    facebook: 'https://fb.me/e/5DhHuB1F5',
-    discord: 'https://discord.com/channels/702801010426511373/713559003661336607',
+    location: 'In Person!',
+    description: 'Intro to Coding Interviews/Arrays \n More details soon.',
```
Comment (entrohpy) [inline src/data.js:24]: The `\n` is extraneous. it doesn't do anything. Would just remove it.

Comment (entrohpy) [reply src/data.js:24]: same for all `\n` below.

File: src/data.js [hunk 3]
```diff
@@ -1,58 +1,71 @@
...
-    day: 'Thu',
-    date: 'Feb. 25',
-    time: '6:30 pm - 9:30 pm PT',
-    location: 'Zoom',
-    description: 'TBA',
-    facebook: 'https://fb.me/e/ddXe8iFSC',
-    discord: 'https://discord.com/channels/702801010426511373/765803471621718086',
+    name: 'Advanced Training #1',
+    day: 'TBA',
+    date: '',
```
Comment (entrohpy) [inline src/data.js:32]: while redundant, would make date 'TBA' as well just so that it looks better in the UI.

File: src/data.js [hunk 4]
```diff
@@ -1,58 +1,71 @@
...
     image: '/static/event/intro.png',
     name: 'Beginner Competitive Programming',
-    quarter: 'Winter',
+    quarter: 'Fall',
     description: 'Continue your journey into competitive programming with Beginner CP. Continuing on from Intro CP, we explore slightly more advanced yet beginner-friendly themes in competitive programming. Whether you are completely new to CP, an expert looking for a quick review, or just an algorithm and problem-solving enthusiast, we promise you will find the workshops interesting and fun!',
   },
   {
     image: '/static/event/interview.png',
-    name: 'Coding Interview Crash Course (Interview Track)',
-    quarter: 'Fall/Winter',
```
Comment (entrohpy) [inline src/data.js:49]: I think we should keep the `Winter` tag?

File: src/data.js [hunk 5]
```diff
@@ -61,37 +74,25 @@ const allEvents = [
...
-    quarter: 'Fall',
-    description: 'From climbing coding leaderboards to competing in ICPC regionals to constructing clever algorithms with clever friends, the Competitive Programming journey starts here. Write your first contest solution and learn what it takes to start solving challenging and often beautiful problems!',
-  },
-  {
-    image: '/static/event/bruinquest.png',
-    name: 'Hack x ICPC x Cyber presents BruinQuest',
-    quarter: 'Fall 2020',
-    description: 'Explore a virtual UCLA campus and complete fun challenges to compete for points, prizes, and the glory of being Bruin Quest Champion!',
-  },
+  // {
```
Comment (entrohpy) [inline src/data.js:77]: just delete these commented events

File: src/data.js [hunk 6]
```diff
@@ -100,35 +101,35 @@ const allEvents = [
...
-    position: 'Officer',
-    description: 'I’m a Junior studying CS at UCLA. At ACM ICPC, I’ve taught Interview Track for the past couple of quarters. Apart from coding, you can catch me reading books, playing FIFA, watching soccer and working out at the gym!',
-    funFacts: {
-      'ice cream': 'Strawberry',
-      professor: 'Darwiche',
-      editor: 'Vim',
-      'TV show in the last year': 'Parks and Rec',
-    },
-  },
+  // {
```
Comment (entrohpy) [inline src/data.js:104]: delete. we can always find these in the git commit history

File: src/pages/events.js [hunk 7]
```diff
@@ -17,7 +17,8 @@ import {
 } from 'react-icons/fa';
 import Container from 'components/container';
 
-import { upcomingEvents, allEvents } from 'data';
+import { upcomingEvents, allEvents, pastEvents } from 'data';
+// import { pastEvents } from '../data';
```
Comment (entrohpy) [inline src/pages/events.js:21]: delete the commented line

**Review by entrohpy — CHANGES_REQUESTED** [review]
Nitpicky stuff. Haven't taken a close look at the content but I think it was fine. Would leave content check to @jacob14916.

**Comment by entrohpy** [comment]
@gu-ana: I must've updated my bio really late so I'm sorry about that but can you update that as well?

**Review by jacob14916 — APPROVED** [review]

**Pull request merged**